roles/logging/handlers/main.yml

```diff
@@ -14,3 +14,8 @@
 
 - name: restart rsyslog
   service: name=rsyslog state=restarted
+
+- name: restart filebeat
+  service: name=filebeat state=restarted must_exist=false
+  failed_when: false
```

Comment:
why do we need failed_when: false? shouldn't ansible fail if is not able to restart the service?